Dockerfile

```diff
@@ -2,6 +2,17 @@ FROM openshift/jenkins-slave-base-centos7
 
 MAINTAINER AeroGear Team <https://aerogear.org/>
 
+#build args
+ARG KEYSTORE_ALIAS=AndroidDebugKey
+ARG FULL_NAME=AG
+ARG ORG_UNIT=AeroGear
+ARG ORG_NAME=RedHat
+ARG LOCALITY=WaterFord
+ARG PROVINCE=WD
+ARG COUNTRY_CODE=IRL
+ARG STORE_PASS=android
+ARG KEY_PASS=android
```

Comment:
I am not sure if we should make these arguments: STORE_PASS, KEY_PASS and KEYSTORE_ALIAS
AFAIK, it won't work correctly with non-default values.

Reply:
You can pass them as build args when you want to build container with user defined keystore